src/util.rs

```diff
@@ -191,22 +192,16 @@ mod tests {
 
     #[test]
     fn test_filename_replace() {
-        let handling = Some(SpaceHandling::Replace);
-        let processed_filename = process_filename("file with spaces.txt", handling);
-        assert_eq!(processed_filename, "file_with_spaces.txt"); // Check if the space is replaced with underscore
+        let handling = SpaceHandling::Replace;
+        let processed_filename = handling.process_filename("file with spaces.txt");
```

Comment:
```suggestion
        let processed_filename = SpaceHandling::Replace.process_filename("file with spaces.txt");
```

Reply:
Addressed in [c716f33](https://github.com/omerbustun/rustypaste/commit/c716f3333b63f84f81e53f5a5609092215c32d56).